### PR TITLE
Add alerts to baremetal dashboard

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: frontend/packages/metal3-plugin/src/components/dashboard/health-card.tsx [hunk 1]
```diff
@@ -41,8 +48,23 @@ const getHostHealthState = (obj): HostHealthState => {
   }
 };
 
-export const HealthCard: React.FC<HealthCardProps> = ({ obj }) => {
+const filterAlerts = (alerts: Alert[]): Alert[] =>
+  alerts.filter((alert) => _.get(alert, 'labels.hwalert'));
```
Comment (rawagner) [inline frontend/packages/metal3-plugin/src/components/dashboard/health-card.tsx]: @jtomasek can you please let me know what kind of filtering is required here ?

Comment (jelkosz) [reply frontend/packages/metal3-plugin/src/components/dashboard/health-card.tsx]: @honza

**Pull request merged**